tests/testthat/test-tokenize.R

```diff
@@ -90,3 +90,61 @@ test_that("Inline comments do not extend past the closing brace", {
   expect_null(out$get_value("seealso"))
 })
 
+test_that("Line numbers are ok", {
+
+  check_line_nums <- function(block, lines) {
+    for (t in names(lines)) {
+      expect_equal(block_get_tag(block, t)$line, lines[[t]], info = t)
+    }
+  }
+
+  text <-
+"#' @title
```

Comment:
Can you indent this please? Otherwise looks good to go.

Reply:
Just that line? I am not sure how to indent multi-line text. Maybe we should use glue for this?

Reply:
The column number doesn't matter for this code?

Reply:
Right. I guess I could have looked at the rest of the tests.... should be good now....